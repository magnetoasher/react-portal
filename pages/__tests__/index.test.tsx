/** @format */

import { shallow } from 'enzyme';
import React from 'react';

import App from '..';

describe('App', () => {
  it('should render correctly', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});
