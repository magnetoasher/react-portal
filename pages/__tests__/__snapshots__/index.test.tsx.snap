// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App app: render correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Typography))
    variant="h2"
  >
    Portal
  </WithStyles(ForwardRef(Typography))>
  <div
    className="makeStyles-container-2"
  >
    <WithStyles(ForwardRef(Card))
      className="makeStyles-card-3"
    >
      <WithStyles(ForwardRef(CardContent))>
        <WithStyles(ForwardRef(Typography))
          variant="body1"
        >
          You are now logged in :)
        </WithStyles(ForwardRef(Typography))>
        <br />
        <WithStyles(ForwardRef(Button))
          color="primary"
          onClick={[Function]}
          size="large"
          type="submit"
          variant="outlined"
        >
          LOGOUT
        </WithStyles(ForwardRef(Button))>
      </WithStyles(ForwardRef(CardContent))>
    </WithStyles(ForwardRef(Card))>
  </div>
</div>
`;
